content/livesync/quickstart.textile

```diff
@@ -1,7 +1,239 @@
+
+Set up environment variables by copying them from the template:
+
+```[sh]
+ cp env.example env.local
+```
+
+Update your @env.local@ file with the following:
+
+```
+POSTGRES_URL="YOUR_FULL_POSTGRES_URL"
+POSTGRES_URL_NON_POOLING="YOUR_FULL_POSTGRES_URL"
+SESSION_SECRET=somesecret
+NEXT_PUBLIC_ABLY_API_KEY=YOUR_ABLY_API_KEY
+```
+
+Export the environment variables in your shell session:
+
+```[sh]
+export $(grep -s -v "^#" env.local | xargs)
+```
+
+h3(#step-4). Configure the Postgres Integration on Ably Dashboard
+
+The Database Connector watches changes in your database table using the transactional "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html and broadcasts those updates over Ably Channels. There are two ways to host the Database Connector, you can use, either the "Ably hosted":/livesync/database-connector#hosted-with-ably or "self hosted":/livesync/database-connector#self-hosted, for this quickstart we'll use the Ably hosted Database Connector. In the "Ably dashboard":https://ably.com/dashboard, choose the project you wish to use for this quickstart and open the @Integrations@ tab. Update the following options available to you:
+
+|_. Option |_. Description |_. Example |
+| URL | Your Neon Postgres database full URL | postgresql://pguser:pgpass@pghost/pgdb?sslmode=require |
+| Outbox table schema | Schema for the "outbox table":/livesync/outbox-nodes-tables#schema. Use "public" for this quickstart. | public |
+| Outbox table name | Name for the "outbox table":/livesync/outbox-nodes-tables#outbox-table. In this quickstart it is "outbox" | outbox |
+| Nodes table schema | Schema for the "nodes table":/livesync/outbox-nodes-tables#schema. Use "public" for this quickstart. | public |
+| Nodes table name | Name for the "nodes table":/livesync/outbox-nodes-tables#nodes-table. In this quickstart it is "nodes" | nodes |
+| SSL mode | Determines the level of protection provided by the SSL connection. Use the default value (@prefer@) for this quickstart. | required |
+| SSL root certificate | Leave this empty for the quickstart. | |
+| Primary site | The primary data center in which to run the integration rule. | eu-central-1-A |
+
+h3(#step-5). Install the dependencies
+
+Install the required dependencies for the web application, including your Models SDK. Then run the @pnpm run db@ command to run the migrations, creating the database tables and populate them with some demo data:
+
+```[sh]
+pnpm install
+pnpm run db
+```
+
+h3(#step-6). Run local web server
+
+Run the web application locally by running the following command:
+
+```[sh]
+pnpm run dev
+```
+
+h3(#step-7). Try it out!
+
+Open the app "localhost:3000":http://localhost:3000 in at least two browser tabs.
+
+Navigate to a post in both tabs, then in one of the tabs try adding, editing, and removing comments from multiple tabs and see the changes reflected to all users in realtime!
```

Comment:
Worth calling out here that for the original author, it's optimistically updated immediately before it gets confirmed along with all other users receiving it.